### ISSUE TITLE
Absorb existing findbugs pull requests?

### ISSUE THREAD

**Issue by mattnelson** [spotbugs/spotbugs#120]
I did a quick glance of the issues/site and did not see this directly answered so forgive me if I missed it. In order to not ignore contributions on findbugs, are those pulls/issues monitored and addressed in spotbugs?

https://github.com/findbugsproject/findbugs/pulls

**Comment by jsotuyod** [comment]
@mattnelson thanks for your concern. We do monitor the upstream project, but so far we have selectively taken issues / PRs. For instace, we haven't even taken any of my own PRs so far.

Right now, the focus is to be able to ship a 3.1.0 release with better Java 8 / Java 9 support. In doing so, we so far focused solely on such issues. In prticular, we have taken:

* https://github.com/findbugsproject/findbugs/pull/48 - getting Travis to work. The current script is way different than the originally proporsed, but we started from there.
* https://github.com/findbugsproject/findbugs/issues/141 - we investigated the issue, found the root cause in BCEL, got it patched and will be fixed in release 3.1.0
* https://github.com/findbugsproject/findbugs/issues/112 - duplicate of the previous one
* https://github.com/findbugsproject/findbugs/issues/95 - we found an issue in Findbugs handling of synthetic methods (ie: lambdas) which we fixed. We also created a new test harness to allow testing without those annotations to get around the limitations on where they could or could not be placed, to be shipped in 3.1.0.

For future releases (4.0.0 and later) we will take our time and bring outstanding PRs and check pending issues. Without help from the original author of each PR / issue however, it may be tricky in some cases. All help is welcomed.
